agent-install/agent-install.sh

```diff
@@ -74,7 +74,7 @@ Options/Flags:
     -O    The exchange organization id (This flag is equivalent to HZN_ORG_ID)
     -u    Exchange user authorization credentials (This flag is equivalent to HZN_EXCHANGE_USER_AUTH)
     -a    Exchange node authorization credentials (This flag is equivalent to HZN_EXCHANGE_NODE_AUTH)
-    -d    The id to register this node with (This flag is equivalent to NODE_ID or HZN_DEVICE_ID)
+    -d    The id to register this node with (This flag is equivalent to HZN_NODE_ID, NODE_ID or HZN_DEVICE_ID. NODE_ID and HZN_DEVICE_ID are deprecated)
```

Comment:
- We can't deprecate `HZN_DEVICE_ID` until the agent and `hzn` support `HZN_NODE_ID` in all of the places it currently uses `HZN_DEVICE_ID`.
- You are missing an open parentheses.